index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Carrie Short - Web Developer</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="vendor/styles/font-awesome.min.css">
+    <link type="text/css" rel="stylesheet" href="styles/base/reset.css">
+    <link type="text/css" rel="stylesheet" href="styles/base/default.css">
+    <link type="text/css" rel="stylesheet" href="styles/layout.css">
+    <link type="text/css" rel="stylesheet" href="styles/module/header.css">
+    <link type="text/css" rel="stylesheet" href="styles/module/banner.css">
+    <link type="text/css" rel="stylesheet" href="styles/module/portfolio-preview.css">
+    <link type="text/css" rel="stylesheet" href="styles/state/hover.css">
+    <link type="text/css" rel="stylesheet" href="styles/state/sticky.css">
+    <script id="preview-template" type="text/x-handlebars-template">
+        <article class="project-preview" data-project="{{project}}" style="background-image:url(img/projects/{{previewImage}})">
+          <section>
+            <h3>{{title}}</h3>
+            <p>{{shortDesc}}</p>
+          </section>
+        </article>
+    </script>
+    <script id="detail-template" type="text/x-handlebars-template">
+        <h3>{{title}}</h3>
+        <img src="img/projects/{{previewImage}}" />
+        <p class="completed">{{publishedOn}}</p>
+        <p class="description">{{{description}}}</p>
+        <a class="close"><i class="fa fa-times-circle"></i></a>
+    </script>
+  </head>
+  <body>
+    <div class="main-container">
+      <header>
+        <h1><a href="#" data-link="overview">Carrie <span></span> Short</a></h1>
+        <nav>
+          <a href="#" class="mobile-menu"><i class="fa fa-bars"></i></a>
+          <ul>
+            <li data-link="about"><a href="#">About</a></li>
+            <li data-link="history"><a href="#">History</a></li>
+            <li data-link="skills"><a href="#">Skills</a></li>
+            <li data-link="portfolio"><a href="#">Portfolio</a></li>
+            <li><a href="https://github.com/CarrieShort"><i class="fa fa-github"></i></a></li>
+          </ul>
+        </nav>
+      </header>
+      <section class="banner" data-content="overview">
+        <ul>
+          <li>UX/UI</li>
+          <li>Visual Design</li>
+          <li class="next-line">HTML</li>
```

Comment:
Refactor selection to use nth-child selectors, so it's easier to manipulate for different resolutions.